ee/clickhouse/migrations/0026_persons_and_groups_on_events.py

```diff
@@ -0,0 +1,35 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.replication.utils import clickhouse_is_replicated
+from ee.clickhouse.sql.events import EVENTS_TABLE_JSON_MV_SQL, KAFKA_EVENTS_TABLE_JSON_SQL
+from posthog.client import sync_execute
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+ADD_COLUMNS_BASE_SQL = """
+ALTER TABLE {table}
+ON CLUSTER '{cluster}'
+ADD COLUMN IF NOT EXISTS person_id UUID,
+ADD COLUMN IF NOT EXISTS person_properties VARCHAR,
+ADD COLUMN IF NOT EXISTS group0_properties VARCHAR,
+ADD COLUMN IF NOT EXISTS group1_properties VARCHAR,
+ADD COLUMN IF NOT EXISTS group2_properties VARCHAR,
+ADD COLUMN IF NOT EXISTS group3_properties VARCHAR,
+ADD COLUMN IF NOT EXISTS group4_properties VARCHAR
+"""
+
+
+def add_columns_to_required_tables(_):
+    sync_execute(ADD_COLUMNS_BASE_SQL.format(table="events", cluster=CLICKHOUSE_CLUSTER))
+
+    if clickhouse_is_replicated():
+        sync_execute(ADD_COLUMNS_BASE_SQL.format(table="writable_events", cluster=CLICKHOUSE_CLUSTER))
+        sync_execute(ADD_COLUMNS_BASE_SQL.format(table="sharded_events", cluster=CLICKHOUSE_CLUSTER))
+
+
+operations = [
+    migrations.RunPython(add_columns_to_required_tables),
+    migrations.RunSQL(f"DROP TABLE events_json_mv ON CLUSTER '{CLICKHOUSE_CLUSTER}'"),
+    migrations.RunSQL(f"DROP TABLE kafka_events_json ON CLUSTER '{CLICKHOUSE_CLUSTER}'"),
```

Comment:
I assume this is a table we aren't currently using, otherwise won't this cause us to lose events?